terraform/modules/lambda/versions.tf

```diff
@@ -1,4 +1,4 @@
 
 terraform {
-  required_version = "~> 0.12.9"
+  required_version = ">= 0.13"
```

Comment:
as of 0.14 and above this should include a `required_providers` block like:

```hcl
terraform {
  required_version = ">= 0.13"
  required_providers {
    aws = {
      source = "hashicorp/aws"
    }
  }
}
```

Reply:
Okay, this is definitely to be handled.